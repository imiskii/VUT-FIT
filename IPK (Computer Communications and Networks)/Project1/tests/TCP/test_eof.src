HELLO
SOLVE (+ 3 3)
SOLVE (* 2 3)