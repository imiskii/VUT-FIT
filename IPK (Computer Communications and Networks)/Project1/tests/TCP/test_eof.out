HELLO
RESULT 6
RESULT 6
BYE

